client/app/components/about-modal/about-modal.html

```diff
@@ -0,0 +1,6 @@
+<a href="#"  ng-click="info.open()" >
+    About
```

Comment:
can we add the `translate` attribute to this element? or do something like `{{ About | translate}}`

point being, all hardcoded text gotta support translations